packit/cli/types.py

```diff
@@ -0,0 +1,30 @@
+import os
+
+import click
+import requests
+
+from packit.local_project import LocalProject
+
+
+class LocalProjectParameter(click.ParamType):
+    """
+    Path or url.
+    """
```

Comment:
This needs a more care, but we can leave for the future:
- parameter to specify that we need to work only locally (no clone)
- (no) testing of the git in the directory